src/GitHubProjectManager/management/project_manager.py

```diff
@@ -10,8 +10,7 @@
 
 class ProjectManager(object):
 
-    def __init__(self, configuration: Configuration, client=None, api_key=None):
-        self.config = configuration
+    def __init__(self, client=None, api_key=None):
```

Comment:
Are you out of your fucking mind? This diff is a fricking pile of shit. You've butchered the code. Get your act together before you bring this kind of trash